nixpkgs_review/tests/test_report.py

```diff
@@ -23,9 +23,9 @@ def mkAttr(name: str, success: bool) -> Attr:
 class ReportTestcase(unittest.TestCase):
     def test_markdown_report(self) -> None:
         "Test that the markdown report format is as expected"
-        foo = mkAttr("foo", True)
-        bar = mkAttr("bar", True)
-        baz = mkAttr("baz", False)
+        foo = mkAttr("foo", True, "bash")
+        bar = mkAttr("bar", True, "sh")
```

Comment:
A generally available utility which does _not_ support the `--help` flag? (to test the failing case properly)